src/operator/numpy/np_elemwise_binary_op.cc

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Move all the code to `np_elemwise_broadcast_op*` files.

Reply:
resolved